app/views/tasks/_tasks.html.slim

```diff
@@ -116,9 +110,8 @@
       = form_for :comment, url: task_comments_path(task), remote: true do |f|
         .write-comment
           .input-group
-            = f.label_tag 'new_comment_label', t('comments.new_comment'), class: 'input-group-addon'
+            = f.label 'new_comment_label', t('comments.new_comment'), class: 'btn btn-light disabled input-group-addon'
```

Comment:
Is this really a disabled button? What about using an [input-group with an `input-group-text`](https://getbootstrap.com/docs/5.3/forms/input-group/)?